test/linux/eepromtool/eepromtool.c

```diff
@@ -456,7 +456,7 @@ int main(int argc, char *argv[])
    }
    else
    {
-      ec_adaptert * adapter = NULL;
+      ec_adaptert * adapter, head = NULL;
```

Comment:
This doesn't compile, because head is not a pointer to ec_adaptert. Prefer not to declare more than one variable per line to avoid this type of mistake.

Reply:
Thank you for the review. How embarrassing, I was sure I'd tested this beforehand. I've updated the PR to declare `head` on a separate line.